content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 ---
 
 <!-- overview -->
-This page describes common concepts in the Kubernetes API.
-
+This page describes Kubernetes' API and conventions for interacting with it.
 
 <!-- body -->
 The Kubernetes API is a resource-based (RESTful) programmatic interface provided via HTTP. It supports retrieving, creating,
-updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET), includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. It also supports efficient change notifications on resources via "watches" and consistent lists to allow other components to effectively cache and synchronize the state of resources.
+updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET). The API includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. Kubernetes also supports efficient change notifications on resources via "watches" and consistent lists, so that API clients can effectively cache, track, and synchronize the state of resources.
+
+## Kubernetes API terminology {#standard-api-terminology}
+
+Most Kubernetes API resource types are [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects): they represent a concrete instance of a concept on the cluster, such as a {{< glossary_tooltip term_id="pod" >}} or {{< glossary_tooltip term_id="namespace" >}}. A smaller number of API resource types are "virtual" - they often represent operations rather than objects, such as a permission check (use a POST with a JSON-encoded body of kind `SubjectAccessReview` to the `subjectaccessreviews` resource). All objects have a unique [names](/docs/concepts/overview/working-with-objects/names/#names) to allow idempotent creation and retrieval, but virtual resource types may not have unique names if they are not retrievable or do not rely on idempotency.
```

Comment:
Why do we keep adding this glossary_tooltip everywhere?
If a user is already looking into this page, and he/she wants to learn more about the API, how likely he/she doesn't know what is a Pod or what is a namespace?

Reply:
Everyone has to start somewhere @tengqm. What I _don't_ know is the gaps in every reader's knowledge.

I haven't seen tooltips as unhelpful. Is there a disadvantage?

Reply:
I need to look at the rendered page to determine if *this* page needs a tooltip about pods and namespaces.
There are plenty of other pages that engage the reader about `Pod` and `Namespace`.

I am more interested in knowing what "virtual" means in this context?

Reply:
`SubjectAccessReview` looks like good example of a virtual resource, according to the existing definition. This page is the first place I came across that wording.

Reply:
I've omitted this change from the latest revision.